build.xml

```diff
@@ -589,7 +589,8 @@
           <dependency groupId="com.github.jbellis" artifactId="jamm" version="${jamm.version}"/>
           <dependency groupId="org.yaml" artifactId="snakeyaml" version="1.26"/>
           <dependency groupId="junit" artifactId="junit" version="4.12" />
-          <dependency groupId="org.mockito" artifactId="mockito-core" version="3.2.4" />
+          <dependency groupId="org.mockito" artifactId="mockito-core" version="3.7.7" />
```

Comment:
I had to increase the version as this version supports mocking of static methods (with use of Powermock I tried too but it was failing for various reasons), here we have static mocks available nicely. The first version supporting this is 3.4.0 but why not to bump it to the latest one if we are doing it anyway.

https://wttech.blog/blog/2020/mocking-static-methods-made-possible-in-mockito-3.4.0/